modules/pel/peltool/peltool.py

```diff
@@ -307,6 +307,23 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        if not os.path.isdir(args.list):
+            sys.exit(f"{args.list} is not a valid directory")
+        print("This function is under development")
+        final_summary={}
+        for root, _, files in os.walk(args.list):
+            for file in files:
+                if args.extension and args.extension != os.path.splitext(file)[1]:
```

Comment:
are we conveying this extension argument details as part of the list option usage? or could we use python argparse magic?